folium/folium.py

```diff
@@ -578,6 +578,10 @@ def choropleth(self, geo_path=None, geo_str=None, data_out='data.json',
             keyword argument will enable conversion to GeoJSON.
         reset: boolean, default False
             Remove all current geoJSON layers, start with new layer
+        smooth_factor: float, default None
+            How much to simplify the polyline on each zoom level. More means
+            better performance and smoother look, and less means more accurate
+            representation. Leaflet defaults to 1.0.
```

Comment:
@JamesGardiner I was just thinking out loud about the docs. Let's not worry about it in this PR.